pkg/workflow/task/bootstrap/bootstrap.go

```diff
@@ -26,7 +27,12 @@ type Bootstrapper interface {
 	) (*types.Cluster, error)
 
 	// DeleteBootstrapCluster deletes a local cluster created with CreateCluster.
-	DeleteBootstrapCluster(ctx context.Context, cluster *types.Cluster, isUpgrade bool) error
+	DeleteBootstrapCluster(
+		ctx context.Context,
+		cluster *types.Cluster,
+		operationType constants.Operation,
```

Comment:
Sorry, can you remind me what this is/for again?

Reply:
Its the existing interface for the bootstrapper. This change doesn't seek to refactor that. I did, however, have to update to the latest interface to get it working.